Explorer/Assets/DCL/MarketplaceCredits/MarketplaceCreditsMenuController.cs

```diff
@@ -324,6 +325,13 @@ private async UniTaskVoid CheckForSidebarButtonStateAsync(CancellationToken ct)
 
                 var creditsProgramProgressResponse = await marketplaceCreditsAPIClient.GetProgramProgressAsync(ownProfile.UserId, ct);
                 SetSidebarButtonState(creditsProgramProgressResponse);
+
+                if (!creditsProgramProgressResponse.HasUserStartedProgram())
+                {
+                    // Open the Marketplace Credits panel by default when the user didn't start the program and has landed in Genesis City.
+                    await UniTask.WaitUntil(() => realmData.IsGenesis(), cancellationToken: ct);
```

Comment:
Will it await forever if the realm is not "genesis"? maybe a timeout could be used, or add the realm check in the confitional like `if (realmData.IsGenesis() && !creditsProgramProgressResponse.HasUserStartedProgram())`